roles/ssh_hardening/tasks/hardening.yml

```diff
@@ -1,6 +1,6 @@
 ---
```

Comment:
ther still seem to be single quotes (`'`) here, that have been changed in the other files